drivers/pinmux/stm32/pinmux_stm32l4x.h

```diff
@@ -70,9 +70,13 @@
 	 STM32_OSPEEDR_VERY_HIGH_SPEED)
 #define STM32L4X_PINMUX_FUNC_PA7_ADC12_IN12 STM32_MODER_ANALOG_MODE
 
+#define STM32L4X_PINMUX_FUNC_PA9_I2C1_SCL                                      \
+	(STM32_PINMUX_ALT_FUNC_4 | STM32_OPENDRAIN_PULLUP)
```

Comment:
Note that some stm32l4 have PA9, PA10 with AF for I2C1 but not all in the series : for example not valid on stm32L476.